src/components/user-profile.html

```diff
@@ -77,27 +34,41 @@
         margin-bottom: 25px;
       }
 
-      input[type=text],
-      select {
+      .icon {
+        fill:#6495ED;
+      }
+
+      .iconContainer {
```

Comment:
rather than setting a class, just style the icon element itself
```suggestion
      jha-icon-circle-plus-outline {
```

Reply:
This class is just a container that I'm using to position my icons in my UI, not the icons themselves? Not sure what you mean here? 
Also I removed the .icon class that I had coloring my icons and just tried
 icon { fill: #6495ED }; but it didn't take? Do the jha-icons read as icon elements for CSS?

Reply:
@frankielb they should be read like any other element in the template (here's an example from Reports).

I guess I was thinking a container shouldn't be necessary for the icon but I'm not entirely sure why at this point.